common/app/layout/Sublink.scala

```diff
@@ -48,7 +48,7 @@ object Sublink {
       EditionalisedLink.fromFaciaContent(faciaContent),
       faciaContent.card.cardStyle,
       faciaContent.card.mediaType,
-      pillar.name.toLowerCase(),
+      Pillar(storyContent).map(_.name).getOrElse("").toLowerCase,
```

Comment:
It would be nice to add the function `.map(_.name).getOrElse("").toLowerCase` to `Pillar`